api/users/service/userService.go

```diff
@@ -0,0 +1,82 @@
+package services
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/null-channel/eddington/api/users/models"
+	repositories "github.com/null-channel/eddington/api/users/repositories"
+)
+
+type IUserService interface {
```

Comment:
I general feel like this interface is adding more complexity without solving any complexity, I'm ok leaving, but in general I don't add interfaces till later when they are needed.